integrations/kuberay/prometheus_metadata.yaml

```diff
@@ -0,0 +1,96 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
```

Comment:
If possible I would recommend launch as HIDDEN first, test that everything works (metric detection, everything displays correctly, links are configured correctly), before setting launch_stage as GA

Reply:
Especially if you need to coordinate with a docs writer on the docs corresponding to the documentation.yaml

Reply:
Changed to HIDDEN.